ci/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3'
 services:
 
   postgres:
-    image: postgres
+    image: mdillon/postgis
```

Comment:
This is the official postgis image. It should be a pure extension of the base postgres image, and it seems like it is safe as a drop-in replacement.

Reply:
Nice.